### PR TITLE
Italian translation

### PR DESCRIPTION
**Pull request by OB1Ken0b1**
Started translation to Italian
Fixed typos in the English files

### CODE REVIEW

**Comment by ntnhon** [comment]
Hello, thank you for correcting the typo and syntax errors in English version 👍

File: it-Italian/Localizable.strings [hunk 1]
```diff
@@ -0,0 +1,298 @@
...
+// Verification
+"enter_otp" = "Inserisci OTP";
+"enter_activation_code" = "Inserisci codice di verifica";
+"paste_and_verify" = "Incolla & verifica";
+"code_found_from_the_clipboard" = "\"%@\" trovato nella clipboard";
+"wrong_code_too_many_times" = "Codice errato inserito troppe volte";
+"send_new_activation_code" = "Ti invieremo un nuovo codice di verifica a \"%@\"";
+"check_new_verification_code" = "Controlla la tua email per il nuovo codice di verifica";
+// Time
+<strind name="created_on" = "Creato";
```
Comment (ntnhon) [inline it-Italian/Localizable.strings]: Syntax error here.

Comment (OB1Ken0b1) [reply it-Italian/Localizable.strings]: thanks, that's fixed now. The same issue was on the original EN files, so I've amended there too.

Comment (ntnhon) [reply it-Italian/Localizable.strings]: thank you!

Comment (ntnhon) [reply it-Italian/Localizable.strings]: That's because I started with the .xml file and do some find/replace to make the .strings file 😅

**Comment by OB1Ken0b1** [comment]
That's the iOS strings translated to Italian. It took much longer than I expected! :) 
Maybe you can use the find/replace again to make the xml version?

**Comment by ntnhon** [comment]
Wow thank you bravo! Such a great work! I know it isn't easy to translate the FAQ part. I will take care of the rest :)

**Comment by OB1Ken0b1** [comment]
No worries at all, happy to help.
It would be good to see the translation in action, maybe in a dev environment. Some things were difficult to translate without knowing the context or where the words would be used. Also, the sentence length may be an issue and not displayed properly, so I may need to replace some of these with something else that fits better.

**Comment by ntnhon** [comment]
Absolutely! But as I also have a full-time job, it will take a little bit of time.

**Comment by ntnhon** [comment]
Hi, I created the `strings.xml` file based on your work: https://github.com/simple-login/Mobile-Apps-Translation/commit/9983b23729bc74b2ddc9fe6744d598eb79972f37
However you forgot to translate 2 keys: `faq_4_question` & `faq_4_answer`. Can you please finish it? Thank you

**Pull request merged**